tests/nemo_text_processing/zh/data_text_normalization/test_cases_math.txt

```diff
@@ -1,3 +1,2 @@
 78:96~七十八比九十六
--23℃~负二十三摄氏度

```

Comment:
why this is deleted?

Reply:
The removed item seems like the input/output for one of our classes, <measure>, which is accountable for width, heights, and etc. I thought temperature should also be in the <measure>'s file.